_includes/header.html

```diff
@@ -1,7 +1,7 @@
 <header class="site-header">
     <div class="grid-x grid-padding-x">
         <div class="cell small-12">
-            <h3 class="logo"><a href="{{ "/" | relative_url }}">Extension Workshop</a></h3>
+            <h3 class="logo"><a href="{{ "/" | relative_url }}">Firefox Extension Workshop</a></h3>
```

Comment:
@muffinresearch Should this be reverted as well if we're not changing the header?